x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -276,7 +276,8 @@ export class AlertingAuthorization {
 
   public async getFindAuthorizationFilter(
```

Comment:
Should we rename this to `getAuthorizationFilter` if we're now expanding it to more general purpose filter?

Reply:
fixed: dcea9504cf03d767eae3f8a778e2d85a759b64a5